package.json

```diff
@@ -16,6 +16,8 @@
 	"private": true,
 	"devDependencies": {
 		"@eslint/js": "^9.7.0",
+		"@types/eslint__js": "^8.42.3",
+		"@types/minimist": "^1.2.5",
```

Comment:
minimist should be with zx, and what is `eslint__js`?

Reply:
for some reason zx doesn't ship minimist's types 🤷 
the eslint__js is types for the eslint configs
https://www.npmjs.com/package/@types/eslint__js